tests/format/test_FormatNXmxDLS16M.py

```diff
@@ -18,22 +18,22 @@
 @pytest.mark.parametrize(
     "master_h5",
     [
-        "/dls/i04/data/2019/cm23004-1/20190109/Eiger/gw/Thaum/Thau_4/Thau_4_1_master.h5",
-        "/dls/i04/data/2019/cm23004-1/20190109/Eiger/gw/Thaum/Thau_4/Thau_4_1.nxs",
+        "/dls/i04/data/2021/cm28182-4/TestProteinaseK/protK3/protK3_1_master.h5",
```

Comment:
How much work would it be to move away from hidden, internal test datasets with this PR, or are we basically stuck with that for the DLS-related format classes?